src/profile-logic/call-tree.js

```diff
@@ -287,7 +287,12 @@ function _getInvertedStackSelfTimes(
   sampleCallNodes: Array<IndexIntoCallNodeTable | null>,
   interval: Milliseconds
 ): {
+  // In an inverted profile, the root node keeps all the self time
```

Comment:
Probably would be clearer if we had "root nodes", plural. How about:
`// In an inverted profile, all the self time is accounted to the root nodes."

Reply:
yeah this sounds good